apps/workflow-cf.yml.j2

```diff
@@ -55,11 +57,15 @@ Resources:
           {% endif %}
         JobRoleArn: !Ref TaskRoleArn
         ExecutionRoleArn: !GetAtt ExecutionRole.Arn
+        NetworkConfiguration:
+          AssignPublicIp: ENABLED
```

Comment:
A public IP is required for tasks to have internet access in our typical deployments using the default VPC. It may not be necessary in Earthdata Cloud environments using a private VPC.

```suggestion
        {% if security_environment != 'EDC' %}
        NetworkConfiguration:
          AssignPublicIp: ENABLED
        {% endif %}
```

Reply:
Did an ad-hoc deployment into the edc uat account and things worked fine. The help text when creating a job definition suggests this setting is required for outbound internet access, regardless of the VPC's network configuration:

> ![Screenshot from 2023-07-22 15-17-45](https://github.com/ASFHyP3/hyp3/assets/17994518/917404fa-3252-49a9-9ffc-22295226d55e)